### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by samujjal1618**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/cplusplus/map/index.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ size of map is 1
 Inserting data with insert member function.
 
 ```c++
-myMap.insert(make_pair("earth", 1));
-myMap.insert(make_pair("moon", 2));
+myMap.insert(pair<char, int> ("earth", 1));
```
Comment (RandellDawson) [inline guide/english/cplusplus/map/index.md]: This change is not necessary.   The original use of `make_pair` is valid syntax.

File: guide/english/cplusplus/map/index.md [hunk 2]
```diff
@@ -67,8 +67,8 @@ size of map is 1
...
+myMap.insert(pair<char, int> ("moon", 2));
```
Comment (RandellDawson) [inline guide/english/cplusplus/map/index.md]: This change is not necessary.   The original use of `make_pair` is valid syntax.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Please revert the changes on the lines specified.  The typo correction is fine.

**Comment by thecodingaviator** [comment]
@RandellDawson This has gone stale

**Comment by RandellDawson** [comment]
Hi @samujjal1618 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**